source/compiler/sc.h

```diff
@@ -727,7 +727,7 @@ SC_FUNC cell do_static_check(int use_warning);
 SC_FUNC void pushstk(stkitem val);
 SC_FUNC stkitem popstk(void);
 SC_FUNC void clearstk(void);
-SC_FUNC int plungequalifiedfile(char *name);  /* explicit path included */
+SC_FUNC int plungequalifiedfile(char *name,char new_extensions);  /* explicit path included */
```

Comment:
```suggestion
SC_FUNC int plungequalifiedfile(char *name,int new_extensions);  /* explicit path included */
```
In this codebase boolean values are usually stored in variables of type `int`. Not sure why, but still...